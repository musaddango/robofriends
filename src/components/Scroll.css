.scroll{overflow:scroll; 
		height:500px; 
		border:1px solid gray;
		}
.scroll::-webkit-scrollbar {
	    display: none;
		}