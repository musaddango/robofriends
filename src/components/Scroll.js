const Scroll = (props) => {
  console.log(props.children)
  return (
    <div style={{ overflow: 'scroll', border: '5px solid black', height: '450px'}}>
      {props.children}
    </div>
  );
};

export default Scroll;
