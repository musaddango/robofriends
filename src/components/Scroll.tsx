import React from "react";

type Props = {
  children?: JSX.Element
}

const Scroll: React.FC<Props> = (props) => {
  return (
    <div style={{ overflow: 'scroll', border: '5px solid black', height: '450px'}}>
      {props.children}
    </div>
  );
};

export default Scroll;
