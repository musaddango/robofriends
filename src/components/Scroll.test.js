import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Scroll from './Scroll';

it('expect to render Card component', ()=>{
    const wrapper = shallow(<Scroll />)

    expect(toJson(wrapper)).toMatchSnapshot();
});