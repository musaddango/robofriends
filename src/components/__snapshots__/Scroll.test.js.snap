// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  style={
    Object {
      "border": "5px solid black",
      "height": "450px",
      "overflow": "scroll",
    }
  }
/>
`;
